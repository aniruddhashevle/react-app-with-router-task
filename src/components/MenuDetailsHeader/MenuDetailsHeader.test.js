import React from 'react';
import { MenuDetailsHeader } from '.';

it('MenuDetailsHeader renders correctly', () => {
    const wrapper = shallow(
        <MenuDetailsHeader details={{ title: 'test title', subTitle: 'test sub-title' }} />
    )
    expect(wrapper).toMatchSnapshot();
});