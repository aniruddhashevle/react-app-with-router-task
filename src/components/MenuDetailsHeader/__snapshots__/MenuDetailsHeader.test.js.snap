// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuDetailsHeader renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MenuDetailsHeader
    details={
      Object {
        "subTitle": "test sub-title",
        "title": "test title",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="title"
        >
          test title
        </h2>,
        <p
          className="description"
        >
          test sub-title
        </p>,
        <WithStyles(Divider) />,
      ],
      "className": "menu-details-header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "test title",
          "className": "title",
        },
        "ref": null,
        "rendered": "test title",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "test sub-title",
          "className": "description",
        },
        "ref": null,
        "rendered": "test sub-title",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="title"
          >
            test title
          </h2>,
          <p
            className="description"
          >
            test sub-title
          </p>,
          <WithStyles(Divider) />,
        ],
        "className": "menu-details-header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "test title",
            "className": "title",
          },
          "ref": null,
          "rendered": "test title",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "test sub-title",
            "className": "description",
          },
          "ref": null,
          "rendered": "test sub-title",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
